tests/favorites.test.js

```diff
@@ -20,18 +24,26 @@ const schema = {
       type: 'string'
     },
     favs: {
-      type: 'string'
+      type: 'array',
+      items: {
+        type: 'object'
+      }
     },
     grocery_id: {
       type: 'string'
     }
   },
   required: ['ingredient_id', 'user_id', 'favs', 'grocery_id']
 }
-const example = { 'ingredient_id': '4aacafd5-6106-11e9-9ffc-a3e52b2d927f',
-  'user_id': '4aacafd2-6106-11e9-9ffc-a3e52b2d927f',
-  'favs': 'desc for department1',
-  'grocery_id': '4aac88cf-6106-11e9-9ffc-a3e52b2d927f'
+
+const testId = __generateId();
+const favoriteIngredients = generateArrWithId(ingredients, 'ingredient_id');
+
+const example = {
+  'ingredient_id': testId,
```

Comment:
all of this ids are related to different entities - they cannot be the same varialbe. this should be re done

Reply:
sure, the template before had the same id for all of them that's why I kept it like this. changing them now